aws/scripts/py/origin_change.py

```
@@ -100,9 +130,11 @@ def update_distribution_configs(distribution_ids, distribution_configs):
 def main():
     print("Starting main function...")
     distribution_ids = fetch_distributions_ids()
-    distribution_configs = fetch_distributions_configs(distribution_ids)
+    filtered_distribution_ids, distribution_configs = fetch_and_filter_distributions(
+        distribution_ids
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)